cogboard-webapp/src/components/Widget/helpers.js

```diff
@@ -23,27 +21,10 @@ export const renderCardContent = (
   status,
   expandContent,
   expanded,
-  handleExpandClick
+  handleToggle
 ) => {
-  return (
-    <StyledCardContent>
-      {content && content.errorMessage ? (
-        <ErrorMessage {...content} />
-      ) : !disabled && !expandContent ? (
-        <WidgetContent id={id} type={type} content={content} />
-      ) : expandContent ? (
-        type === 'AemHealthcheckWidget' ? (
-          <StyledStatusIconButton href={content.url} target="_blank">
-            <StatusIcon status={status} size="large" />
-          </StyledStatusIconButton>
-        ) : (
-          <StyledIconWrapper>
-            <StatusIcon status={status} size="large" />
-          </StyledIconWrapper>
-        )
-      ) : (
-        'Disabled'
-      )}
+  const WidgetFooter = () => {
```

Comment:
Don't define React component in other components or render functions. It is serious overkill for performance!